frontend/bower.json

```diff
@@ -9,8 +9,8 @@
     "toastr": "^2.1.2",
     "font-awesome": "~4.5.0",
     "jquery": "1.11.1",
-    "alpaca": "1.5.14",
     "handlebars": "3.0.3",
+    "alpaca": "https://github.com/HalcyonChimera/alpaca.git",
```

Comment:
The alpaca dependency should be omitted entirely in favor of having it implicitly included as a dependency of ember-cli-dynamic-forms, which should be updated to v0.2.3